docs/concepts/projects/dependencies.md

```diff
@@ -591,6 +594,15 @@ The `dev` group is special-cased; there are `--dev`, `--only-dev`, and `--no-dev
 inclusion or exclusion of its dependencies. Additionally, the `dev` group is
 [synced by default](#default-groups).
 
+To install dependencies you can use install or sync
+
+```console
+$ uv python install
+$ uv sync --all-extras # installs all optional dependencies
+$ uv sync --group lint # installs dependency group lint
+$ uv sync --only-group lint # installs ONLY dependency group lint
```

Comment:
These two would make more sense in the following "Dependency groups" section, e.g., after

> Once groups are defined, the `--group`, `--only-group`, and `--no-group` options can be used to
include or exclude their dependencies.